pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "python-fixturify-project"
-version = "0.1.3"
```

Comment:
I think this is done by [`poetry version` during release](https://github.com/scalvert/python-fixturify-project/blob/main/RELEASE.md#releasing-a-new-version), right?

Reply:
Yes. Will revert this